qiskit/quantum_info/operators/symplectic/clifford.py

```diff
@@ -110,93 +118,245 @@ def __init__(self, data, validate=True):
+        "and will be removed no sooner than 3 months after the releasedate. "
+        "Use Clifford.paulis method instead.",  # pylint:disable=bad-docstring-quotes
+    )
     def destabilizer(self, value):
         """Set the value of destabilizer block of the StabilizerTable"""
         inds = slice(0, self.num_qubits)
-        self._table.__setitem__(inds, value)
+        self._paulis.__setitem__(inds, value)
+
+    def tableau(self, x=True, z=True, stabilizer=True, destabilizer=True):
+        """Return the tableau of the Clifford.
+
+        Args:
+            x (bool): if True, it returns x component.
+            z (bool): if True, it returns z component.
+            stabilizer (bool): if True, it returns stablizer component.
+            destabilizer (bool): if True, it returns destabilizer component.
+
+        Returns:
+            np.ndarray: the tableau.
+
+        """
+        if stabilizer and destabilizer:
+            indices = slice(None)
+        elif stabilizer:
+            indices = slice(self.num_qubits, 2 * self.num_qubits)
+        elif destabilizer:
+            indices = slice(0, self.num_qubits)
+        else:
+            return np.array([], dtype=bool)
+
+        if x and z:
+            return np.hstack((self.paulis.x[indices], self.paulis.z[indices]))
+        elif x:
+            return self.paulis.x[indices].copy()
+        elif z:
+            return self.paulis.z[indices].copy()
+        else:
+            return np.array([], dtype=bool)
+
+    def stabilizer_tableau(self):
+        """Return the tableau of the Clifford's stabilizer.
+
+        Returns:
+            np.ndarray: the tableau.
+        """
+        return self.tableau(destabilizer=False)
+
+    def destabilizer_tableau(self):
+        """Return the tableau of the Clifford's destabilizer.
+
+        Returns:
+            np.ndarray: the tableau.
+        """
+        return self.tableau(stabilizer=False)
+
+    def stabilizer_x_tableau(self):
+        """Return the tableau of the Clifford's stabilizer x components.
+
+        Returns:
+            np.ndarray: the tableau.
+        """
+        return self.tableau(z=False, destabilizer=False)
+
+    def stabilizer_z_tableau(self):
+        """Return the tableau of the Clifford's stabilizer z components.
+
+        Returns:
+            np.ndarray: the tableau.
+        """
+        return self.tableau(x=False, destabilizer=False)
+
+    def destabilizer_x_tableau(self):
+        """Return the tableau of the Clifford's stabilizer x components.
+
+        Returns:
+            np.ndarray: the tableau.
+        """
+        return self.tableau(z=False, stabilizer=False)
+
+    def destabilizer_z_tableau(self):
+        """Return the tableau of the Clifford's stabilizer z components.
+
+        Returns:
+            np.ndarray: the tableau.
+        """
+        return self.tableau(x=False, stabilizer=False)
+
+    def stabilizer_list(self):
```

Comment:
Another alternative here would be just `stabilizers(self)` and `destabilizers(self)`, but i'm ok either way

Reply:
I renamed to stabilizers and destabilizers.